examples/redux/src/redux/modules/commits.js

```diff
@@ -0,0 +1,44 @@
+import * as Repo from './repos';
+import * as Users from './users';
+import { normalize } from '../../../../../src';
+
+export const STATE_KEY = 'commits';
+
+export default function reducer(state = {}, action) {
+  switch (action.type) {
+    case Action.ADD_COMMITS:
+      return {
+        ...state,
+        ...action.payload
+      };
+
+    default:
+      return state;
+  }
+}
+
+const Action = {
+  ADD_COMMITS: 'ADD_COMMITS'
+};
+
+export const addCommits = (commits = {}) => ({
+  type: Action.ADD_COMMITS,
+  payload: commits
+});
+
+export const getCommits = ({ page = 0 } = {}) => (dispatch, getState, { api, schema }) => {
+  const state = getState();
+  const owner = Repo.selectOwner(state);
+  const repo = Repo.selectRepo(state);
+  return api.repos.getCommits({
+    owner,
+    repo
+  }).then((response) => {
+    const data = normalize(response, [ schema.commit ]);
+    dispatch(Users.addUsers(data.entities.users));
+    dispatch(addCommits(data.entities.commits));
```

Comment:
Triggering multiple actions here may be considered bad practice. I think the general consensus is that it's better to trigger a single action that maps to some user interaction (e.g. `LOAD_COMMITS`) in order to avoid inconsistent intermediate states and unnecessary re-renders.